src/components/Footer/styles.module.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../styles/custom_variables.scss";
+
+footer {
+	border-top: 1px solid $grey-mid;
```

Comment:
S   p   a    c    i     n   g       looks a bit  off here.

Reply:
Ahh, yeah it moved to 4 spaces when i copied and pasted, I'll fix it.